### ISSUE TITLE
Python package question

### ISSUE THREAD

**Issue by Connor-Bowley** [MathieuLeclercq/SlicerJawSegmentation#5]
If I recall, you opted for using the system python environment so you could get newer packages than were available in Python 3.6. With the update of Slicer's (not SlicerSALT's yet) Python to 3.9.10, can you get everything you need into the Slicer python environment?

**Comment by MathieuLeclercq** [comment]
I am working on Python 3.9.7 so it should be possible to get everything I need into the Slicer Python environment when it updates to 3.9.10.

**Comment by Connor-Bowley** [comment]
The Slicer mainline has update to  Python 3.9.10, and SlicerSALT will at some point. After talking with @bpaniagua we think you should update your extension to be compatible with the mainline slicer (i.e. get everything into the Slicer Python environment and build against the head of Slicer). Once that is done, we can merge the extension index PR. Your extension won't be able to get put into SlicerSALT until SlicerSALT updates to the head of Slicer as well, but we need to do that anyway.

So long story short, you are good to start on this.

**Comment by MathieuLeclercq** [comment]
Thank you. I am going to be working on that.

**Comment by Connor-Bowley** [comment]
How is this coming?

**Comment by MathieuLeclercq** [comment]
I managed to install all required dependencies. I Still need to fix some errors due to the installation of a newer version of PyTorch that causes some minor incompatibilities. I will get back to you when everything is done.

**Comment by MathieuLeclercq** [comment]
The module is now fully working on PythonSlicer.
